contributing.md

```diff
@@ -459,13 +459,32 @@ coverage report. You'll need to have [lcov](http://ltp.sourceforge.net/coverage/
 
 1. In your `build` folder, compile with `-DCMAKE_BUILD_TYPE=Coverage`
 
-        cmake -DCMAKE_BUILD_TYPE=Coverage ../
+    If using a colcon workspace:
+
+        cd <path to colcon workspace>
+        colcon build --cmake-args -DCMAKE_BUILD_TYPE=Coverage
+
+    If using plain cmake:
+
+        cd <path to build directory>
+        cmake -DCMAKE_BUILD_TYPE=Coverage ..
         make
 
 1. Run a single test, or all the tests
 
-       ./workspace/build/package_name/bin/UNIT_TestName_TEST   (single test)
-        make test                                          (all tests)
+    If using a colcon workspace:
+
+       # single test
+       ./build/<package_name>/bin/UNIT_<TestName>_TEST
+
+       # all tests
+       cd build/<package_name>
+       make test
```

Comment:
This works, but I think that when using colcon, this is cleaner:

```
        cd <path to colcon workspace>
        colcon test --packages-select <package_name>
```

This way the logs end up in the `logs` directory.

Reply:
ok, I'll update that. Is there a `colcon` command to run all tests? The command you provided only runs one single test?

Reply:
Here's a quick cheat sheet:

Run all tests for all packages in the workspace:

```
colcon test
```

Run all tests for a single package:

```
colcon test --packages-select <package_name>
```

Run specific tests:

```
colcon test --ctest-args -R <part of test name>
# or
./build/<package_name>/bin/<TestName>
```